### PR TITLE
Documentation Cleanup for 0.28 Release

### PR DESCRIPTION
**Pull request by blink1073**
- Update the Xkcd tutorial and use watch mode (corresponds with https://github.com/jupyterlab/jupyterlab_xkcd/tree/0.28).
- Clean up the handling of watch mode
- Add more docs for watch mode

### CODE REVIEW

**Comment by blink1073** [comment]
cc @parente

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -196,7 +196,15 @@ To have the system build after each source file change, run:
 jupyter lab --dev-mode --watch
 ```
 
-and refresh the browser.
+and refresh the browser.  You can open another terminal and run
+
+```bach
```
Comment (ian-r-rose) [inline CONTRIBUTING.md]: bach -> bash

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ You can also use the installer from the [Node.js](https://nodejs.org) website.
 
 JupyterLab requires Jupyter Notebook version 4.3 or later.
 
-If you use ``conda``, you can install it as:
+If you use ``conda``, you can install notebook as:
```
Comment (ian-r-rose) [inline CONTRIBUTING.md]: "install notebook using"

File: CONTRIBUTING.md (hunk 1 above)
Comment (blink1073) [reply CONTRIBUTING.md]: I prefer baroque highlighting

Comment (ian-r-rose) [reply CONTRIBUTING.md]:
```handel
echo "Messiah"
```

**Pull request merged**